src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.SignedNumber.cs

```diff
@@ -0,0 +1,306 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Diagnostics;
+
+namespace System.Text.Json
+{
+    public ref partial struct Utf8JsonWriter
+    {
+        /// <summary>
+        /// Writes the property name and <see cref="long"/> value (as a JSON number) as part of a name/value pair of a JSON object.
+        /// </summary>
+        /// <param name="propertyName">The UTF-16 encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
```

Comment:
While you know that `System.String` is UTF-16(LE), and I know that `System.String` is UTF-16(LE); but most people just think of it as "a string" (right up until surrogates pop in).  I'd leave the notion of encoding out while it's typed as a string.  (After all, we could internally store it as UTF-32BE, but expose it via the indexer as UTF-16LE)

Reply:
And to reinforce myself: The encoding is about when it's in byte form.  As a string or char it's in "concept form", meaning there's no encoding.  (I'm sure the Unicode spec has a word for the "concept form", but I didn't look it up).